site/docs/components/country-symbol/usage.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Country Symbol
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### Using the Country Symbol component
+
+To reduce bundle size, Country Symbols are in a separate package that you should first install using:
+
+`npm install @salt-ds/countries` or `yarn add @salt-ds/countries`
+
+You can then import each Country Symbol component using its country code as follows:
+
+`import { GB } from "@salt-ds/countries";`
```

Comment:
not sure it makes sense to have import here and mentioned again bellow. Maybe merge the two somehow?